wallpaperdownloader/README.md

```diff
@@ -0,0 +1,17 @@
+# WallpaperDownloader snap
+
+This project creates a working snap of WallpaperDownloader application.
+WallpaperDownloader is a simple GUI Java based application to download wallpapers from the Internet and manage them. 
+For more information, please visit the official [Wiki](https://bitbucket.org/eloy_garcia_pca/wallpaperdownloader/wiki/Home) of the project.
+
+## Current state
+
+Working features:
+ - The applications is working fine. You can even change the downloads folder (by default inside snap directory) to a directory within the home's user.
```

Comment:
`application`

`user's home folder` - maybe? I can't tell what you are trying to get at here